### PR TITLE
chapter3: Mostly static page

### PR DESCRIPTION
**Pull request by phinguyenphuoc**
image

### CODE REVIEW

**Comment by phinguyenphuoc** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield :title %> | Ruby on Rails Tutorial Sample App</title>
```
Comment (chien-pv) [inline app/views/layouts/application.html.erb]: dùng i18n em

**Comment by chien-pv** [comment]
commented

**Comment by phinguyenphuoc** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**